doc/man3/OPENSSL_s390xcap.pod

```diff
@@ -34,14 +34,16 @@ There are three types of tokens:
 The name of a processor generation. A bit in the environment variable's
 mask is set to one if and only if the specified processor generation
 implements the corresponding instruction set extension. Possible values
-are z900, z990, z9, z10, z196, zEC12, z13, z14 and z15.
+are B<z900>, B<z990>, B<z9>, B<z10>, B<z196>, B<zEC12>, B<z13>, B<z14>
+and B<z15>.
```

Comment:
Actually, I would quote those values, or use `C<` rather than `B<`

Reply:
Why?  When we do the commands we have ```B<DER>|B<PEM>``` and when we talk about the API we use ```B<int64_t>``` for types, and look at the bolded flags in ```ASYNC_start_job.pod``` for example.  Seems to me that these are literals, and should be bold.

Reply:
I am with @richsalz on this.

Reply:
Okie